fenix/app/src/main/java/org/mozilla/fenix/compose/ListItemTabLarge.kt

```diff
@@ -138,7 +141,7 @@ fun ListItemTabSurface(
     onClick: (() -> Unit)? = null,
     tabDetails: @Composable () -> Unit,
 ) {
-    var modifier = Modifier.size(328.dp, 116.dp)
+    var modifier = Modifier.size(ITEM_WIDTH.dp, ITEM_HEIGHT.dp)
     if (onClick != null) modifier = modifier.then(Modifier.clickable { onClick() })
```

Comment:
Unrelated, but mentioning as var could cause [perf issues](https://medium.com/androiddevelopers/jetpack-compose-stability-explained-79c10db270c8)

```suggestion
    val modifier = Modifier.size(ITEM_WIDTH.dp, ITEM_HEIGHT.dp)
    ..clickable(enabled = onClick != null) { onClick!!() }
```

Could actually inline it in the `Card` param

Reply:
Created [bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1836979) for this

Reply:
thanks!